application/models/Project_keyword_model.php

```diff
@@ -3,7 +3,6 @@
 class Project_keyword_model extends MY_Model {
 
     
-
```

Comment:
Random unnecessary white space change?

Reply:
Struggled with your white space issues for about 90 minutes today. Did discover I could get an add-on for gedit (my coding editor) that would show white space, but even then still struggled to figure out what git diff is showing up. I don't think I'm cut out to be a white space detective somehow. I've tried my hardest, but may not have succeeded.